bottomless/src/replicator.rs

```diff
@@ -192,7 +192,7 @@ impl Default for Options {
         Options {
             create_bucket_if_not_exists: true,
             verify_crc: true,
-            use_compression: CompressionKind::None,
+            use_compression: CompressionKind::Gzip,
```

Comment:
How is this change related to auto checkponiting?

Reply:
I think, I've left it when started running comparisons with Litestream. Keeping non-compressed WAL frames around is extremely expensive and we definitely shouldn't do that by default.